packages/interactive-messages/src/adapter.ts

```diff
@@ -24,10 +25,10 @@ function formatMatchingConstraints<C extends AnyConstraints>(matchingConstraints
   }
 
   let ret: AnyConstraints = {};
-  if (typeof matchingConstraints === 'string' || matchingConstraints instanceof RegExp) {
-    ret.callbackId = matchingConstraints;
+  if (!isPlainObject(matchingConstraints)) {
```

Comment:
out of curiosity, which test was failing? its a shame we couldn't axe this dependency 😿

Reply:
There were a few tests that would pass a `number` (and some other invalid types) to `actions()` and `options()` to make sure it threw a `TypeError`. RIP 💀

Reply:
darn!